tests/app/shared/title-editor/title-editor.component.spec.ts

```diff
@@ -0,0 +1,78 @@
+import { CUSTOM_ELEMENTS_SCHEMA, DebugElement } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { FormControl, FormGroup, FormsModule } from '@angular/forms';
+import { By } from '@angular/platform-browser';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { Apollo } from 'apollo-angular';
+
+import { SharedModule } from '../../../../src/app/shared/shared.module';
+import { TitleEditorComponent } from '../../../../src/app/shared/title-editor/title-editor.component';
+
+describe('CommentEditor', () => {
+  let fixture: ComponentFixture<TitleEditorComponent>;
+  let debugElement: DebugElement;
+  let component: TitleEditorComponent;
+
+  const TEST_INITIAL_TITLE = 'abc';
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      declarations: [TitleEditorComponent],
+      imports: [FormsModule, SharedModule, BrowserAnimationsModule],
+      providers: [Apollo],
```

Comment:
As this component is isolated, there is no longer any dependencies on Apollo so this portion is not needed.